cmd/ncproxy/main.go

```diff
@@ -50,7 +67,12 @@ func main() {
 	if conf.NodeNetSvcAddr != "" {
 		log.G(ctx).Debugf("connecting to NodeNetworkService at address %s", conf.NodeNetSvcAddr)
 
-		opts := []grpc.DialOption{grpc.WithInsecure()}
+		// Register views to collect data.
+		if err := view.Register(ocgrpc.DefaultClientViews...); err != nil {
```

Comment:
I don't think we need this, don't use any stats stuff anywhere.

Reply:
KK wasn't sure. Just followed the example they gave

Reply:
Wait, do we just not want the views registered or the interceptor at all?

Reply:
We need the GRPC handler registered, just not the `view.Register` thing.

Reply:
Done